diff --git a/setup_libuv.py b/setup_libuv.py
index 2ad70fb..315de78 100644
--- a/setup_libuv.py
+++ b/setup_libuv.py
@@ -142,7 +142,6 @@ class libuv_build_ext(build_ext):
 
         if sys.platform.startswith('linux'):
             self.compiler.add_library('dl')
-            self.compiler.add_library('rt')
 
             self.compiler.add_include_dir(
                 os.path.join(self.libuv_dir, 'compat', 'linux'))
